src/cli/console.js

```diff
@@ -3,7 +3,7 @@
 const Promise = require('bluebird');
 const _ = require('lodash');
 const chalk = require('chalk');
-const Table = require('cli-table2');
+const Table = require('cli-table5');
```

Comment:
Shouldn't this be a 3 instead of a 5? Seems to be the reason that Travis fails.

Reply:
Ah thanks, fixed it.